tests/playwright-test/playwright-test-fixtures.ts

```diff
@@ -194,7 +195,7 @@ type Fixtures = {
 };
 
 const common = base.extend<CommonFixtures>(commonFixtures as any);
-export const test = common.extend<Fixtures>({
+export const test = common.extend<ServerFixtures>(serverFixtures as any).extend<Fixtures>({
```

Comment:
why do we need this?

Reply:
Because we now run server in playwright tests.